b2sdk/sync/policy.py

```diff
@@ -29,7 +29,29 @@ class AbstractFileSyncPolicy(object):
     DESTINATION_PREFIX = NotImplemented
     SOURCE_PREFIX = NotImplemented
 
-    def __init__(self, source_file, source_folder, dest_file, dest_folder, now_millis, args):
+    NEWER_FILE_SKIP = 101
```

Comment:
Are the ints for speed?  Strings would make debugging easier.

Reply:
We should be using enums, but 2.7 doesn't have them. ints are faster, but not that much. Django coders get used to using those due to the necessity of putting the enum in a db column, where speed matters way more than here. I'll fix it.

Reply:
Ok.